setup.cfg

```diff
@@ -1,11 +1,14 @@
+[aliases]
+test=pytest
+
 [flake8]
 max-line-length = 79
 exclude =
   versioneer.py,
   lander/_version.py
 
 [tool:pytest]
-addopts = --flake8
+addopts = --flake8 --cov=documenteer
```

Comment:
As above, I think we can drop the addopts here.

Reply:
See response above.